stumpy/core.py

```diff
@@ -121,7 +121,13 @@ def norm_func(Q, T, A_norm=None, other_norm=None, normalize=True, p=2.0):
         The desired z-normalized/non-normalized function (or class)
     """
     if exclude is None:
-        exclude = ["normalize", "p", "T_A_subseq_isconstant", "T_B_subseq_isconstant"]
+        exclude = [
+            "normalize",
+            "p",
+            "T_A_subseq_isconstant",
+            "T_B_subseq_isconstant",
+            "T_subseq_isconstant_func",
```

Comment:
I added `T_subseq_isconstant_func` here since, in some functions, we have param `T_subseq_isconstant_func`

Reply:
hmmm, I don't think that is great. I think it should be added to the individual function decorators that have `T_subseq_isconstant_func`

The parameters that I have here are the ubiquitous ones

Reply:
> The parameters that I have here are the ubiquitous ones

Exactly. That was my concern too. Okay...I will revise it. Thanks for the input!